app/src/main/java/org/glucosio/android/tools/GlucosioConverter.java

```diff
@@ -23,7 +23,7 @@
 import java.math.BigDecimal;
 import java.math.RoundingMode;
 
-public class GlucosioConverter {
+public abstract class GlucosioConverter {
```

Comment:
ah it was final when I created this PR, but if it doesn't need to instantiate, blocking and preventing it would be more better to everyone who would use this class :)